kathara-rest/kathara-rest.dockerfile

```diff
@@ -25,6 +25,8 @@ ENV PYTHONPATH="/kathara-rest/"
 
 WORKDIR /kathara-rest/src
 
+ENV LAB_DIR="/labs"
+VOLUME /labs
```

Comment:
Added Volume `/labs` where the generated laboratories are saved on the REST container, independent from the source code.